tests/compute/test_shuffle.py

```diff
@@ -0,0 +1,112 @@
+import pytest
+import numpy as np
+
+from typing import Callable
+
+from numpy.testing import assert_array_almost_equal, assert_array_equal
+
+
+import roughpy.compute as rpc
+
+
+
+
+@pytest.fixture
+def random_shuffle(rng) -> Callable[[int, int], rpc.ShuffleTensor]:
```

Comment:
Where does `rng` come from in the fixture? Also, should this have a `_` prefix as in `_word_to_idx_fn` below?

Reply:
rng is a global fixture defined in conftest.py. It's just an instance of np.random.default_rng with a seed. The naming is just to indicate that this is a helper function for these tests only, and not something that is needed elsewhere. Fixtures, by convention, are named with no leading underscore. Pytest itself only considers fixtures from the local scope and those explicitly declared as shared. (Such as those in conftest.py.)